packages/wdio-allure-reporter/tests/api.test.ts

```diff
@@ -35,121 +61,198 @@ describe('reporter reporter api', () => {
     })
 
     it('should support end step', () => {
-        reporter.endStep('passed')
+        AllureReporter.endStep(Status.PASSED)
         expect(process.emit).toHaveBeenCalledTimes(1)
-        expect(process.emit).toHaveBeenCalledWith(events.endStep, 'passed')
+        expect(process.emit).toHaveBeenCalledWith(events.endStep, Status.PASSED)
     })
 
     it('should support addStep without attachment', () => {
-        reporter.addStep('foo')
+        AllureReporter.addStep('foo')
         expect(process.emit).toHaveBeenCalledTimes(1)
-        const step = { 'step': { 'status': 'passed', 'title': 'foo' } }
+        const step = { 'step': { 'status': Status.PASSED, 'title': 'foo' } }
         expect(process.emit).toHaveBeenCalledWith(events.addStep, step)
     })
 
     it('should support default step status for addStep', () => {
-        reporter.addStep('foo', { content: 'baz' } )
+        AllureReporter.addStep('foo', { content: 'baz' } )
         expect(process.emit).toHaveBeenCalledTimes(1)
-        const step = { 'step': { 'attachment': { 'content': 'baz', 'name': 'attachment', 'type': 'text/plain' }, 'status': 'passed', 'title': 'foo' } }
+        const step = { 'step': { 'attachment': { 'content': 'baz', 'name': 'attachment', 'type': 'text/plain' }, 'status': Status.PASSED, 'title': 'foo' } }
         expect(process.emit).toHaveBeenCalledWith(events.addStep, step)
     })
 
     it('should throw exception for incorrect status for addStep', () => {
         // @ts-expect-error invalid param
-        const cb = () => { reporter.addStep('foo', { content: 'baz' }, 'invalid-status')}
-        expect(cb).toThrowError('Step status must be passed or failed or broken or canceled or skipped. You tried to set "invalid-status"')
+        const cb = () => { AllureReporter.addStep('foo', { content: 'baz' }, 'invalid-status')}
+        expect(cb).toThrowError('Step status must be failed or broken or passed or skipped. You tried to set "invalid-status"')
     })
 
     it('should throw exception for incorrect status for endStep', () => {
         // @ts-expect-error invalid param
-        const cb = () => { reporter.endStep('invalid-status') }
-        expect(cb).toThrowError('Step status must be passed or failed or broken or canceled or skipped. You tried to set "invalid-status"')
+        const cb = () => { AllureReporter.endStep('invalid-status') }
+        expect(cb).toThrowError('Step status must be failed or broken or passed or skipped. You tried to set "invalid-status"')
     })
 
     it('should pass correct data from addArgument', () => {
-        reporter.addArgument('os', 'osx')
+        AllureReporter.addArgument('os', 'osx')
         expect(process.emit).toHaveBeenCalledTimes(1)
         expect(process.emit).toHaveBeenCalledWith(events.addArgument, { name: 'os', value: 'osx' })
     })
 
+    describe('step', () => {
+        it('should add custom step', async () => {
```

Comment:
please add all possible things: `links`, `parameter`, `description`, `descriptionHtml` in separate tests.

Reply:
seems it will be nice to move it to separate spec file

Reply:
Missed fix?

Reply:
Yep, will provide it today

Reply:
👍